conf.d/fzf.fish

```diff
@@ -3,7 +3,7 @@ if not set --query fzf_fish_custom_keybindings
     # \cf is Ctrl+f
     bind \cf '__fzf_search_current_dir'
     bind \cr '__fzf_search_history'
-    bind \cv '__fzf_search_shell_variables'
+    bind \cv '__fzf_search_shell_variables (set --show (set --names) | psub) (set --names)'
```

Comment:
You can probably just save that into a variable and then re-use it below.

Reply:
Then we will have an extra variable shown. And it doesn't save much.

```
> time set --names
________________________________________________________
Executed in  645.00 micros    fish           external
   usr time  394.00 micros  394.00 micros    0.00 micros
   sys time  240.00 micros  240.00 micros    0.00 micros
```

Reply:
Does it lead to an extra variable? I don't care about saving time but not having two copies of the string `'__fzf_search_shell_variables (set --show (set --names) | psub) (set --names)'`

Reply:
> '__fzf_search_shell_variables (set --show (set --names) | psub) (set --names)'

Oh, this can be avoided. I'm taking about the binded function.

Reply:
As explained before, `set --show (set --names)` shows `$history` but `set --show` doesn't.